File: core/examples/market-data/search_markets.ts
import pmxt from '../../src';

const main = async () => {
    const poly = new pmxt.Polymarket();
    const kalshi = new pmxt.Kalshi();

    console.log('Polymarket:', await poly.fetchMarkets({ query: 'Fed' }));
    console.log('Kalshi:', await kalshi.fetchMarkets({ query: 'Fed' }));
};

main();

File: sdks/python/examples/market-data/search_markets.py
import pmxt

def main():
    poly = pmxt.Polymarket()
    kalshi = pmxt.Kalshi()

    print('Polymarket:', poly.fetch_markets(query='Trump'))
    print('Kalshi:', kalshi.fetch_markets(query='Trump'))

if __name__ == "__main__":
    main()
